llvm/test/Transforms/LoopVectorize/AArch64/sve-epilog-vect.ll

```diff
@@ -11,7 +11,7 @@ target triple = "aarch64-linux-gnu"
 
 ; DEBUG: LV: Checking a loop in 'main_vf_vscale_x_16'
 ; DEBUG: Create Skeleton for epilogue vectorized loop (first pass)
-; DEBUG: Main Loop VF:vscale x 16, Main Loop UF:2, Epilogue Loop VF:vscale x 8, Epilogue Loop UF:1
+; DEBUG: Main Loop VF:vscale x 16, Main Loop UF:2, Epilogue Loop VF:8, Epilogue Loop UF:1
```

Comment:
Also, I think it's probably worth adding a variant of the first RUN line where we set the flag `-sve-prefer-fixed-over-scalable-if-equal=false` just to defend the selection of scalable VFs for the epilogue.